src/modules/tls/tls_domain.c

```diff
@@ -146,19 +156,17 @@ static void setup_dh(SSL_CTX *ctx)
 		return;
 	}
 
-#if (OPENSSL_VERSION_NUMBER >= 0x1010000fL) && !defined(LIBRESSL_VERSION_NUMBER)
-	/* libssl >= v1.1.0 */
-	DH_set0_pqg(dh, p, NULL, g);
```

Comment:
In the PR description you mention `OpenSSL >= 1.1.1`, now the comments mention  `OpenSSL 1.1.0+`, so I am just asking for confirmation it is ok to remove the use of `DH_set0_pqg` here. If yes, then it will be merged.

Reply:
Yes ok to remove: OpenSSL 1.1.0 also has the macro SSL_CTX_set_dh_auto.